sympy/printing/tests/test_latex.py

```diff
@@ -149,6 +157,7 @@ def test_latex_symbols():
     assert latex(Symbol('omega') ** Symbol('beta')) == r"\omega^{\beta}"
 
 
+
```

Comment:
Thanks, I saw that. It's not really clear about what is recommended for trivial commits, though. It states that "the best way to make a change is to add a new commit" but then it also mentions rebasing and editing the commit history. But I'll go ahead and amend the commit, a separate commit for an empty line seems really not useful.